ui/src/Modals/NewLibrary/Index.jsx

```diff
@@ -84,10 +87,24 @@ function NewLibraryModal(props) {
       };
 
       dispatch(newLibrary(data));
-
-      close();
     }
-  }, [close, dispatch, mediaType, name, selectedFolders]);
+  }, [dispatch, mediaType, name, selectedFolders]);
+
+  //checks if the modal is open then you redirect.
```

Comment:
comment not formatted properly. Remove the new line after the comment, and add a space at the start of the comment (after the two slashes)